yarn-project/end-to-end/src/e2e_nested_contract.test.ts

```diff
@@ -114,10 +114,13 @@ describe('e2e_nested_contract', () => {
       ];
 
       const tx = await new BatchCall(wallet, actions).send().wait();
-      const logs = L2BlockL2Logs.unrollLogs(await wallet.getUnencryptedLogs(tx.blockNumber!, 1)).map(log =>
-        toBigIntBE(UnencryptedL2Log.fromBuffer(log).data),
-      );
-      expect(logs).toEqual([20n, 40n]);
+      const filter = {
+        fromBlock: tx.blockNumber!,
+        limit: 2, // 2 logs expected
```

Comment:
Is this `limit` used anywhere? I can't see that it is.

Reply:
It was a stale param. Removed that in https://github.com/AztecProtocol/aztec-packages/pull/2600/commits/c2127bf383c8677b84af03a5bcfd0b36e65b81ad